base/applications/wordpad/wordpad.c

```diff
@@ -2048,7 +2048,7 @@ static LRESULT OnNotify( HWND hWnd, LPARAM lParam)
 
         sprintf( buf,"selection = %d..%d, line count=%ld",
                  pSC->chrg.cpMin, pSC->chrg.cpMax,
-                SendMessageW(hwndEditor, EM_GETLINECOUNT, 0, 0));
+                (long int)SendMessageW(hwndEditor, EM_GETLINECOUNT, 0, 0));
```

Comment:
Not unsigned, is it because of "%ld" ? If not, why not fixing the formatting string instead?

Reply:
Why is this formatting string even allowed to be like this without translations?

I'll change the formatting string but don't ask me to fix the translations...